containers/Ideas/Phase.js

```diff
@@ -2,6 +2,7 @@ import React from 'react'
 import {View} from 'react-native'
 
 import {TextSourceSans} from '../../components/TextSourceSans'
```

Comment:
maybe a rule to have spaces for curly braces for named imports?

Reply:
I agree, I couldn't find a good one though. Formating with Prettier would also help with that